src/USER-MISC/pair_agni.cpp

```diff
@@ -66,9 +68,24 @@ static const char cite_pair_agni[] =
   " number    = {1},\n"
   " pages     = {511},\n"
   " year      = {2017},\n"
+  "}\n\n"
+  "@article{huan2019jpc,\n"
```

Comment:
There are **far** too many citations here. There should only be citations that describe the method in use, ideally only one citing the most relevant publication. Since you have two variants of the functional form, you could define two citation strings (with the one most relevant publication for each variant) and place the `citeme->add()` call so that only the one variant in use is added. All other related citations can be added to the manual, but should not be included in the code. Thanks.

Reply:
We actually have a publication that compares the 2 functional forms, so I'll just use that to cover both versions.